include/aws/common/byte_order.inl

```diff
@@ -19,7 +19,7 @@
 #include <aws/common/byte_order.h>
 #include <aws/common/common.h>
 
-#ifdef _MSC_VER
+#if defined(_MSC_VER) || defined(__MINGW32__)
```

Comment:
All of these `#if defined(_MSC_VER) || defined(__MINGW32__)` should actually be `#if defined(_WIN32)`

It's easy to mix up compiler/platform when a compiler is platform-specific like msvc. But the mix up does become clear when you see something like this

Reply:
Good point. I've addressed this in the new version of the commit.